website/docs/r/aws_auth_backend_client.html.md

```diff
@@ -68,6 +68,11 @@ The following arguments are supported:
 * `sts_region` - (Optional) Override the default region when making STS API 
     calls. The `sts_endpoint` argument must be set when using `sts_region`.
 
+* `use_sts_region_from_client` - (Optional) If set, overrides both `sts_endpoint`
```

Comment:
Could we also document here that this field will only work with versions >= 1.15?